stdlib/test/python/test_python_object.mojo

```diff
@@ -411,39 +411,47 @@ fn test_none() raises:
 
 fn test_getitem_raises() raises:
     var a = PythonObject(2)
-    with assert_raises(contains="'int' object has no attribute '__getitem__'"):
+    with assert_raises(contains="'int' object is not subscriptable"):
         _ = a[0]
+    with assert_raises(contains="'int' object is not subscriptable"):
+        _ = a[0, 0]
 
     var b = PythonObject(2.2)
-    with assert_raises(
-        contains="'float' object has no attribute '__getitem__'"
-    ):
+    with assert_raises(contains="'float' object is not subscriptable"):
         _ = b[0]
+    with assert_raises(contains="'float' object is not subscriptable"):
+        _ = b[0, 0]
 
     var c = PythonObject(True)
-    with assert_raises(contains="'bool' object has no attribute '__getitem__'"):
+    with assert_raises(contains="'bool' object is not subscriptable"):
         _ = c[0]
+    with assert_raises(contains="'bool' object is not subscriptable"):
+        _ = c[0, 0]
 
     var d = PythonObject(None)
-    with assert_raises(
-        contains="'NoneType' object has no attribute '__getitem__'"
-    ):
+    with assert_raises(contains="'NoneType' object is not subscriptable"):
         _ = d[0]
+    with assert_raises(contains="'NoneType' object is not subscriptable"):
+        _ = d[0, 0]
 
-    var with_get = Python.evaluate(
-        "type('WithGetItem', (), {'__getitem__': lambda self, key: f\"Key:"
-        ' {key}"})()'
+    with_get = Python.evaluate(
+        "type('WithGetItem', (), {'__getitem__': lambda self, key: f'Keys: {',"
```

Comment:
Is there a typo here perhaps?  Internally, in CI I'm seeing an error:

```
Unhandled exception caught during execution: ("f-string: expecting '}'", ('<string>', 1, 69, "type('WithGetItem', (), {'__getitem__': lambda self, key: f'Keys: {', '.join(map(str, key))}' if isinstance(key, tuple) else f'Key: {key}'})()", 1, 70))
```

Reply:
Hmm. Lets avoid f-string. I pushed now using `format`, lmk :)